### ISSUE TITLE
syncoid warning: ZFS resume feature not available on source machine - sync will continue without resume support

### ISSUE THREAD

**Issue by gkrokum** [jimsalterjrs/sanoid#595]
I have two systems running Centos 7.8 and ZFS.  One is my primary storage array and I'm using the other one to backup the primary using syncoid.  I recently starting using sanoid and syncoid.  I'm doing a pull from the backup system and when I run syncoid I get the warning about the resume feature not being available on the source machine.  I have updated the OS and ZFS on both, they are running ZFS 0.8.5.  I'm using a non-root user with zfs allow permissions to run synciod.  I have the same user setup on both systems and use ssh keys between them.

After the updates, zpool is reporting that some features are not enabled on the pool and I am unable to do a zpool upgrade.  I get the following error when I run "zpool upgrade -a":  cannot set property for 'data': invalid argument for this pool operation.
Not sure if that is related, but I've always been able to do so in the past.

Any assistance or ideas would be greatly appreciated.

**Comment by secabeen** [comment]
I believe the first issue you describe is the same as this:
https://github.com/jimsalterjrs/sanoid/issues/554

Not sure about the second.

**Comment by jimsalterjrs** [comment]
Issues with `zpool upgrade` are upstream; nothing we can do about it here. I'm not sure if the issue in the first has been addressed yet; @phreaker0 ?

**Comment by gkrokum** [comment]
Yes, the issue with the resume feature appears to be the same as #554.  I'm seeing a return value of 256 when it runs the sourceresume check.  zpool get -o value -H feature@extensible_dataset $pool returns active for the two zpools on my source server.

I figured out the issue with zpool upgrade.  I had rebooted after an OS update, but did not after updating zfs.  Once I rebooted the systems again, I was able to run a zpool update on both.  Obviously nothing to do with the resume issue.

Thanks for the quick responses!

**Comment by phreaker0** [comment]
@jimsalterjrs @gkrokum it should be the same problem as #554 and it's still open, will look into it

**Comment by slrslr** [comment]
How can i enable this ZFS fast resume feature (reported not to be found in this issue title) if i am on Arch Linux?
